Flow.Launcher.Infrastructure/UserSettings/Settings.cs

```diff
@@ -199,7 +199,7 @@ public bool HideNotifyIcon
         }
         public bool LeaveCmdOpen { get; set; }
         public bool HideWhenDeactive { get; set; } = true;
-        public bool RememberLastLaunchLocation { get; set; }
+        public string LauncherPosition { get; set; } = "RememberLastLaunchLocation";
```

Comment:
I think you have defined the enum? Why not use them?


```suggestion
        public LauncherPositions LauncherPosition { get; set; } = LauncherPositions.RememberLastLaunchLocation;
```

Reply:
applied it.